packages/apps/human-app/server/src/common/utils/cache.ts

```diff
@@ -0,0 +1,3 @@
+export const generateCacheKey = (...parts: (string | number)[]): string => {
```

Comment:
Sorry for confusion in last comment: I meant that implementation is nice, didn't mean to move it to util (esp. taking into account that it can't be shared due to `governance` prefix, so let's please move it back